azure-pipelines/simple_test_framework.yml

```diff
@@ -44,29 +44,28 @@ jobs:
         displayName: "Install TARDIS environment"
 
       - bash: |
-              GIT_LFS_SKIP_SMUDGE=1 git clone $(ref.data.github.url) $(ref.data.home)
-              cd $(ref.data.home)
-              git fetch origin
-              git checkout origin/master
-              git lfs pull --include="atom_data/kurucz_cd23_chianti_H_He.h5" origin
-              git lfs pull --include="atom_data/chianti_He.h5" origin
-              git lfs pull --include="plasma_reference/" origin
```

Comment:
I checked and `plasma_reference` folder is not present in `tardis-refdata`. Removing this unnecessary line.